data/data/aws/route53/base.tf

```diff
@@ -85,6 +85,14 @@ resource "aws_route53_record" "etcd_a_nodes" {
   records = [var.etcd_ip_addresses[count.index]]
 }
 
+resource "aws_route53_record" "bootstrap_a_node" {
+  type    = "A"
+  ttl     = "60"
+  zone_id = "${aws_route53_zone.int.zone_id}"
+  name    = "etcd-bootstrap.${var.cluster_domain}"
+  records = ["${var.bootstrap_ip_address}"]
```

Comment:
`["${var.bootstrap_ip_address}"]` -> `var.bootstrap_ip_address`

Reply:
`["${var.bootstrap_ip_address}"]` -> `[var.bootstrap_ip_address]` didn't realize that the records needed a list.